src/shared/modules/auth/index.js

```diff
@@ -26,7 +26,7 @@ import {
 } from './settings'
 
 export const authRequestForSSO = async selectedSSOProvider => {
-  authLog('Initializing auth redirect request for SSO')
+  authLog('Initialising auth redirect request for SSO')
```

Comment:
heh, the original spelling was correct since we mostly use American spelling here! :-)